arrow-libs/core/arrow-atomic/src/jsMain/kotlin/arrow/atomic/AtomicMap.kt

```diff
@@ -0,0 +1,3 @@
+package arrow.atomic
+
+public actual typealias AtomicMap<K, V> = DefaultAtomicMap<K, V>
```

Comment:
This can actually be regular mutable map, no? 🤔

Reply:
But then I don't know how to implement the operations like `putIfAbsent` ':)

Reply:
Since JS is single threaded, can it not just be `if(!containsKey(k)) put(k, v)`? 🤔

Reply:
Sorry, I didn't explain my problem correctly. The problem is that I don't know how to use `expect`/`actual` in that case. If I do:
```kotlin
public actual typealias AtomicMap<K, V> = HashMap<K, V>
```
then I get the following weird error:
```
Actual typealias 'AtomicMap' has no corresponding expected declaration
The following declaration is incompatible because some supertypes are missing in the actual declaration:
* public final expect class AtomicMap<K, V> : Map<K, V> defined in arrow.atomic in file AtomicMap.kt
```